src/agent/include/unix_daemon.hpp

```diff
@@ -1,3 +1,17 @@
 #pragma once
 
-#include <process_options.hpp>
+#include <string>
+#include <unistd.h>
+#include <vector>
+
+class AgentDaemon
```

Comment:
I would suggest to either rename the file to `agent_daemon` or change the class name to `UnixDaemon`.

Reply:
Done. Renamed class to UnixDaemon.